### PR TITLE
Create Windows Installer

### PR DESCRIPTION
**Pull request by jwrober**
First stab at getting a Windows NSIS package installer working #303 . This installer does work, gets the program installed properly. There are some cosmetic issues to be handled in a separate issue.

### CODE REVIEW

File: data/stdmusic.musicspec [hunk 1]
```diff
@@ -7,7 +7,7 @@ artists = "\
 "
 
 [files]
-;music_menu = ""
+music_menu = "stdmusic/CullamBruce-Lockhart--Dawning_Fanfare.ogg"
```
Comment (lmoureaux) [inline data/stdmusic.musicspec:10]: Unwanted change?

File: CMakeLists.txt [hunk 2]
```diff
@@ -16,9 +16,70 @@ list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
...
+	${MINGW_PATH}/libbrotlidec.dll
+	${MINGW_PATH}/libbrotlicommon.dll
+	${MINGW_PATH}/libKF5Archive.dll
+	${MINGW_PATH}/libFLAC-8.dll
+	${MINGW_PATH}/SDL2_mixer.dll
+	${MINGW_PATH}/SDL2.dll
+	${MINGW_PATH}/Qt5Widgets.dll
+	${MINGW_PATH}/Qt5Network.dll
+	${MINGW_PATH}/Qt5Gui.dll
+	${MINGW_PATH}/Qt5Core.dll
```
Comment (lmoureaux) [inline CMakeLists.txt:78]: Probably want to indent

**Review by lmoureaux — CHANGES_REQUESTED** [review]
Skimming through it it looks good :+1: I have a few style comments though, and I think tabs slipped in. No big deal though.

How do you want me to merge this? Is there a logical separation between the commits or is it best to just squash them?

**Review by jwrober — COMMENTED** [review]
removed extraneous line

File: CMakeLists.txt (hunk 2 above)
Comment (jwrober) [reply CMakeLists.txt]: Fixed

File: data/stdmusic.musicspec (hunk 1 above)
Comment (jwrober) [reply data/stdmusic.musicspec]: Fixed

**Comment by jwrober** [comment]
I think with the last two commit's this is ready to merge once the checks pass. Got rid of the extraneous add about the stdmusic and also got all the tabs removed.

**Review by lmoureaux — APPROVED** [review]

**Pull request merged**